apps/dashboard/src/api/subscribers.ts

```diff
@@ -0,0 +1,28 @@
+import type { IEnvironment, IListSubscribersResponseDto } from '@novu/shared';
+import { getV2 } from './api.client';
+
+export const getSubscribers = async ({
+  environment,
+  cursor,
+  limit,
+  email,
+  phone,
+  subscriberId,
+  name,
+}: {
+  environment: IEnvironment;
+  cursor: string;
+  limit: number;
+  email?: string;
+  phone?: string;
+  subscriberId?: string;
+  name?: string;
+}): Promise<IListSubscribersResponseDto> => {
+  const { data } = await getV2<{ data: IListSubscribersResponseDto }>(
+    `/subscribers?cursor=${cursor}&limit=${limit}&email=${email}&phone=${phone}&subscriberId=${subscriberId}&name=${name}`,
```

Comment:
Nit-picky one: We should build the query conditionally if values are provided to avoit sending empty query params to the API.